libs/community/langchain_community/document_loaders/parsers/audio.py

```diff
@@ -12,6 +12,144 @@
 logger = logging.getLogger(__name__)
 
 
+class AzureOpenAIWhisperParser(BaseBlobParser):
+    """Transcribe and parse audio files.
+
+    Audio transcription is with Azure OpenAI Whisper model.
+    This is different to the Open AI Whisper parser and requires
+    an Azure OpenAI API Key.
+
+    Args:
+        api_key: Azure OpenAI API key
+        chunk_duration_threshold: minimum duration of a chunk in seconds
+            NOTE: According to the OpenAI API, the chunk duration should be at least 0.1
+            seconds. If the chunk duration is less or equal than the threshold,
+            it will be skipped.
+    """
+
+    def __init__(
+        self,
+        api_key: Optional[str] = None,
+        *,
+        deployment_id: str,
```

Comment:
We should try to stay consistent with the parameter naming for an Azure deployment across all the different langchain classes. Maybe we should use `deployment_name` or `azure_deployment`?

Reply:
Now the attribute is called `deployment_name` :)